README.md

```diff
@@ -38,6 +41,12 @@ You can download the latest hugo binary from the [release
 page](https://github.com/gohugoio/hugo/releases). Version 0.54 or later is
 required.
 
+On a Linux 64-bit machine, you can install the latest version of Hugo by:
+```
+wget $(curl -s https://api.github.com/repos/gohugoio/hugo/releases/latest | grep -oP '"browser_download_url": "\K(.*)hugo_extended(.*)Linux-64bit.deb')
```

Comment:
there are prebuilt binaries. That might be preferable to doing a syswide install which also assumes a debian distro.

Reply:
ohh, changing